### PR TITLE
Applied pyupgrade to project

### PR DESCRIPTION
**Pull request by Bartvaderkin**
I saw this in the general chat and was curious.

### CODE REVIEW

File: src/open_inwoner/apimock/views.py [hunk 1]
```diff
@@ -63,7 +63,7 @@
         # make dynamic from url conf?
         prefix = request.build_absolute_uri(f"/apimock/{set_name}/")
 
-        with open(file_path, "r") as f:
+        with open(file_path) as f:
```
[the remark this answers is not in the document]
Comment (Bartvaderkin) [reply src/open_inwoner/apimock/views.py:66]: Oh come on.

[the remark this answers is not in the document]
Comment (Bartvaderkin) [reply src/open_inwoner/apimock/views.py:66]: Path traversal is explicitly checked a few lines above.

[the remark this answers is not in the document]
Comment (Bartvaderkin) [reply src/open_inwoner/apimock/views.py:66]: Path traversal is explicitly checked a few lines above.

**Review by pi-sigma — APPROVED** [review]
Looking good, especially the formatted strings! Did you run the script manually or do you have a githook?

**Comment by Bartvaderkin** [comment]
I just ran it manually for now, it is not that important.

**Review by stevenbal — APPROVED** [review]

**Pull request merged**